app/jobs/calendar_importer/parsers/out_savvy.rb

```diff
@@ -5,46 +5,13 @@
 # Parent parser classes should not be added.
 
 module CalendarImporter::Parsers
-  class OutSavvy < Base
-    NAME = 'OutSavvy'
+  class OutSavvy < LdJson
```

Comment:
I don't think we need an `OutSavvy` importer anymore. The `LdJson` one will just work for it instead, right? At least that's the point of this work to be so that we don't need to care about different providers.

Reply:
see [this ticket](https://github.com/geeksforsocialchange/PlaceCal/issues/1589) - i think at a bare min its nice to have a linkback to the source event and its hard to imagine we could autogenerate this. its prob fine to delete now and remake it when we need it tho.

i think its also nice to autodetect urls we know work? in a future iteration we can poss do some earlier autotedetection of the url, give some contextual advice to get to the right page, etc

Reply:
We're autodetecting already in these changes. We just do it by attempting to fetch events and seeing if it works.

Reply:
And to give backlinks, we have a `url` method we populate with the event's URL which we can use already.

Reply:
Right, but this one doesn't actually override anything, does it? It's just plain old `LdJson` with a label.

Reply:
But it does have `NAME`, `KEY` and `DOMAINS` that are used on the front end

Reply:
i have done as you said. its now in only one parser.